l10n_eu_nace/tests/test_res_partner_industry_eu_nace_wizard.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from unittest.mock import patch
+
+from odoo.tests.common import Form, TransactionCase, new_test_user, users
+
+from .test_en_nace_request_results import (
+    NACE_COMMON,
+    NACE_EN,
+    NACE_ES,
+    NACE_FR,
+    create_response,
+)
+
+MOCK_PATH = (
+    "odoo.addons.l10n_eu_nace.wizard.res_partner_industry_eu_nace_wizard.requests.get"
+)
+
+
+class TestResPartnerIndustryEUNaceWizard(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = new_test_user(
```

Comment:
suggestion: change this to [implicitly change the tests env from here onwards](https://github.com/odoo/odoo/blob/49826056eb675cc13cb9797100c583fa286d651b/odoo/tests/common.py#L283-L286):

```suggestion
        cls.uid = new_test_user(
```

Reply:
Done! Thanks for your review @yajo